__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const { app } = require("../app.js");
+const connection = require("../db/connection.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  connection.end();
+});
+describe("GET api/topics", () => {
```

Comment:
We also want to add another describe block and test for a 404 invalid path error :) You can just test for the status and express should throw and automatic error if you give a path that does not exists